docs/hugo/content/guide/authentication/_index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Authentication in Azure Service Operator"
+linkTitle: "Authentication"
+layout: single
+cascade:
+- type: docs
+- render: always
+---
+
+There are two key topics surrounding authentication in Azure Service Operator: The type of credential, and the credential scope.
+
+## Credential type
+
+Azure Service Operator supports four different styles of authentication today.
+
+1. [Service Principal using a Client Secret](./credential-format.md#service-principal-using-a-client-secret)
+2. [Service Principal using a Client Certificate](./credential-format.md#service-principal-using-a-client-certificate)
+3. [Azure-Workload-Identity](https://github.com/Azure/azure-workload-identity) authentication (OIDC + Managed Identity or Service Principal)
+4. [Deprecated] aad-pod-identity authentication (Managed Identity)
```

Comment:
Don't use `.md` on internal links, and remember that files get rendered as directories by Hugo, so documents at the same level are referenced as `../sibling`

Reply:
The `.md` thing I confirmed, but `../sibling` doesn't seem to work correctly, at least on my local Hugo?

If I use `../credential-format#service-principal-using-a-client-secret`, the link it takes me to is: `http://localhost:1313/azure-service-operator/guide/credential-format#service-principal-using-a-client-secret`, which doesn't exist (note the lack of an `authentication` part in the path

but if I use `./credential-format#service-principal-using-a-client-secret`, it works and I get
`http://localhost:1313/azure-service-operator/guide/authentication/credential-format/#service-principal-using-a-client-secret`

Reply:
That's weird.

Reply:
I just double-checked that all of these links here work in Hugo. My assumption is that the actual filestructure puts the `_index` file alongside the folders so you have:
```
index
credential-format/<files>
credential-scope/<files>
```

So if you want to link from the outer index to a child, it's just `./credential-format`, but if you're _inside_ `credential-format` and you wanna link to another peer, you need to leave the directory first so you end up having to do `../credential-scope`.

Just guessing though.